### ISSUE TITLE
Login form with errors example has insufficient color contrast

### ISSUE THREAD

**Issue by angelikatyborska** [bitcrowd/bitstyles#814]
This is how the login form with errors example looks like:

<img width="953" alt="Screenshot 2024-03-20 at 13 18 08" src="https://github.com/bitcrowd/bitstyles/assets/7852553/f1a9cd7a-09d5-43d2-9989-52f1aba3ddfc">

The error message below the input has the color #FFA940 on a white background. This has a contrast ration of 1.9:1 and fails WCAG contrast checks (source: https://webaim.org/resources/contrastchecker/?fcolor=FFA940&bcolor=FFFFFF)

**Comment by angelikatyborska** [comment]
I just discovered that the default color of inputs in a normal state (with value, not focused) also has insufficient contrast. This is the contrast: https://webaim.org/resources/contrastchecker/?fcolor=8C88A7&bcolor=FFFFFF if fails AA with small fonts. Both WAVE and axe-core detect those inputs as failing contrast checks.

![Screen Shot 2024-07-26 at 11 13 11](https://github.com/user-attachments/assets/a0849e69-4a8e-4979-b4a0-b7a08b51442e)

**Comment by angelikatyborska** [comment]
And the default colors of an active button: https://webaim.org/resources/contrastchecker/?fcolor=361AE9&bcolor=BEB3FF

<img width="178" alt="Screenshot 2024-07-26 at 11 35 50" src="https://github.com/user-attachments/assets/b0c3c3a1-de30-4c50-835d-137fe1b8f357">

I'm running axe-core on our new BE project that is using bitstyles with the default configuration and those contrast issues get reported.
